app/modules/project/routes/project-service.ts

```diff
@@ -21,9 +16,8 @@ export class ProjectService {
         return await projectModel.find({ ID: projectID });
     };
 
-    public update = async (reqBody: Object): Promise<any> => {
-        const data = new projectModel(reqBody);
-        return await data.updateOne(data);
+    public update = async (reqParam: string, reqBody: Object): Promise<any> => {
+        return await projectModel.updateOne({ ID: reqParam }, reqBody);
```

Comment:
You do not need to `await` here, you can just return a promise from here, which would be resolved on the caller.

Reply:
You can remove `await` from everywhere in the file, you can keep `async` though.